python/ray/data/read_api.py

```diff
@@ -2900,6 +2901,54 @@ def from_torch(
     )
 
 
+@PublicAPI
+def from_lancedb(
```

Comment:
let's call it `read_lance`, Lance is a file format.

Reply:
Updated